.github/home.css

```
@@ -0,0 +1,495 @@
+* {
+    box-sizing: border-box;
```

```suggestion
    box-sizing: border-box;
    font-family: Pretendard;
```
글자를 사용하는 부분에 계속 font-family가 있는데
이 부분은 이렇게 전체에 적용하는게 더 효율적으로 보입니다!